test/model/create.test-d.ts

```diff
@@ -60,3 +63,14 @@ db.user.create({
   // will be executed to get the initial value.
   firstName: 'John',
 })
+
+const user = db.user.create({
+  // It is possible to create models with null values
+  lastName: null,
+})
+
+// @ts-expect-error lastName property is possibly null
+user.lastName.toUpperCase()
+
+// @ts-expect-error property 'toUpperCase' does not exist on type 'number'
+user.age?.toUpperCase()
```

Comment:
Shouldn't TypeScript complain about the property being possibly null first? 🤔

Reply:
Not in this case because we are using optional chaining 👍

Reply:
Oh, I see. Thanks for the explanation.